lib/strategies/header_strategy.rb

```diff
@@ -0,0 +1,22 @@
+module Phenotype 
+  class HeaderStrategy
+    attr_reader :header, :header_param, :env
+    def initialize(header: 'Sue')
+      @header = header
+    end
+
+    def get_version(env)
```

Comment:
Is there a specific reason for passing the `env` to the instance method rather than initializing the object with it?